pubsub/cloud-client/README.md

```diff
@@ -29,25 +29,28 @@ For more samples, see the samples in
 
 #### Create a new topic
 ```
-  mvn exec:java -Dexec.mainClass=com.example.pubsub.CreateTopicExample -Dexec.args=my-topic
+  mvn exec:java -Dexec.cleanupDaemonThreads=false \
```

Comment:
The `exec.cleanupDaemonThreads` and the `exec.cleanupDaemonThreads` arguments are the same every time, so we should add them to the pom.xml to avoid making our users type them out. Check out how Dane did it [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/opencensus/pom.xml#L76).